src/Game/Game.ml

```diff
@@ -726,6 +727,137 @@ let asym_gcmds gcmds =
 let read_se se = Se.union (read_gcmds se.se_gdef) (e_vars se.se_ev.ev_expr)
 
 (*i ----------------------------------------------------------------------- i*)
+(* \hd{Hash call (hc) occurences} *)
```

Comment:
Can't we have higher-order functions such that we can define *_hash_calls and *_vars by applying the HOF?